src/components/Header/HeaderHomepage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { ReactComponent as Heart } from '../Design/img/heart.svg'
+import { ReactComponent as Profile } from '../Design/img/user-header.svg'
+import { ReactComponent as Create } from '../Design/img/user-plus.svg'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export default function HeaderHomepage() {
```

Comment:
Again this is more like a Navigation. Also do you really need two Header components? Maybe you can display the different SVG with a tournary.